meetuppizza/tests.py

```diff
@@ -32,7 +32,8 @@ def test_signup_redirects(self):
 
   def test_user_is_created(self):
     self.client.post('/sign_up', self.params)
-    self.assertEqual(1, len(User.objects.all()))
+    user = User.objects.get(username='Bjorn')
+    self.assertFalse(user == None)
```

Comment:
This gets the job done, but `unittest` provides [`assertIsNotNone`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsNotNone) for just this situation. Why is it better? Consider this test class:

```py
class TestTwoKindsOfAssertions(unittest.TestCase):

  def test_boolean_assertion(self):
    self.assertFalse(None == None)

  def test_is_not_none_assertion(self):
    self.assertIsNotNone(None)
```

Here's what happens when I run it:

```
FF
======================================================================
FAIL: test_boolean_assertion (__main__.TestTwoKindsOfAssertions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_example.py", line 6, in test_boolean_assertion
    self.assertFalse(None == None)
AssertionError: True is not false

======================================================================
FAIL: test_is_not_none_assertion (__main__.TestTwoKindsOfAssertions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_example.py", line 9, in test_is_not_none_assertion
    self.assertIsNotNone(None)
AssertionError: unexpectedly None

----------------------------------------------------------------------
Ran 2 tests in 0.000s

FAILED (failures=2)
```

One of these failures gives much more information about what went wrong, which will be much more helpful when Future You does something that breaks this test. 

Here is a table of all the built in `unittest` assertions, which I keep open in a browser tab pretty much all the time: https://docs.python.org/3/library/unittest.html#unittest.TestCase.debug. If you're using a `django.test.TestCase` instead of a `unittest.TestCase`, you also have all the extra assertions described here: https://docs.djangoproject.com/en/1.9/topics/testing/tools/#assertions